server-code/Main.js

```diff
@@ -8,7 +8,7 @@ var scriptProps = PropertiesService.getScriptProperties(); // Google apps script
 /**
  * Serves up the website HTML from Index.html
  * 
- * @return {HtmlOutput} request 
+ * @return {HtmlOutput} 
  */
 function doGet(request) {
```

Comment:
this parameter is never used btw

Reply:
Yes, it never is, but I believe it is a required parameter.

Reply:
Yes, it's just me not knowing anything about web dev.